### PR TITLE
Convert gearman to use upstart

### PR DESCRIPTION
**Pull request by Hwesta**
Not sure if this works with 12.04, but is that an issue?  ```/etc/init.d/gearman-job-server``` stopped working after I updated to 14.04.

### CODE REVIEW

**Comment by mistydemeo** [comment]
Doesn't work in 12.04, alas. :( Doesn't necessarily need to block including this though, especially once we have a release with official 14.04 support.

**Comment by jhsimpson** [comment]
I modified this PR so the restart-services tool should work on both Ubuntu 12.04 and 14.04 now.
I also added in the storage service in the list of services that are restarted.

The Old System V method of starting and stopping services is using /etc/init.d/foo restart.
The Not quite as old Upstart method is to use scripts in /etc/init/ which can be called like 'start foo'.
The compromise between these is to use 'service' as in 'service foo start'.  This will try to run an upstart script but it will use the /etc/init.d/ script if that is all there is.

Documented on https://askubuntu.com/questions/2075/whats-the-difference-between-service-and-etc-init-d

Of course, in Ubuntu 16.04, this is all going to change to systemd.  
https://wiki.ubuntu.com/SystemdForUpstartUsers

File: tools/restart-services [hunk 1]
```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 
+sudo service gearman-job-server stop
```
Comment (Hwesta) [inline tools/restart-services]: Should we stop gearman before we stop the archivematica services that depend on it?

File: tools/restart-services [hunk 2]
```diff
@@ -13,10 +16,11 @@ then
...
-sudo rm /tmp/archivematicaMCP*
+sudo service elasticsearch start
+sudo service gearman-job-server start
 sudo start fits
+sudo service nginx start
+sudo service uwsgi start
 sudo start archivematica-mcp-server
 sudo start archivematica-mcp-client
-sudo apache2ctl restart
+sudo service apache2 start
```
Comment (Hwesta) [inline tools/restart-services]: Do we need to stop apache/nginx/uwsgi, or can we just restart them at the end to reload the new configs?

Comment (mistydemeo) [inline tools/restart-services]: Restarting is fine, I think?

Comment (jhsimpson) [inline tools/restart-services]: I believe there are cases where it is necessary to stop apache, at least. For example, if you want to update the MCP database, and apache is still running, it is possible that there are open database connections, and any sql like 'drop database' will hang.

Comment (Hwesta) [inline tools/restart-services]: Ahh, that makes sense.

**Pull request merged**